INSTALL.md

```diff
@@ -513,5 +496,97 @@ $ git submodule update
 
 Then continue with “Crosscompiling from a released tarball” above.
 
+VII. Compiling on Windows with msys2
+-----------------------------------------
+
+TSC can be compiled on Windows using msys2. The msys2 suite provides a convenient Bash shell for Windows, along with an expanding
+repository of prebuilt packages.
+
+
+### Installing and updating msys2 ###
+
+To begin, download and install msys2 from the [official website][3].
+
+After it's installed run any of the shells and execute:
+
+    $ pacman -Syuu
+
+If the base system components are updated, you will be prompted to close the bash windows manually, go ahead and do so.
+
+Now, run either the `MSYS2 MinGW 32-bit` or the `MSYS2 MinGW 64-bit`, depending on which version you are going to build. Note that you
+can download libraries from one shell for the other, but you cannot build for the other architecture.
+
+Now run update again:
+
+    $ pacman -Syuu
+
+
+### Installing the dependencies ###
+
+Once it's finished, install the dependencies:
+
+    $ pacman -S --needed git bison ruby mingw-w64-x86_64-{toolchain,extra-cmake-modules,ruby,mruby,cegui,sfml,libxml++2.6,gperf}
+
+Or, for 32-bit:
+
+    $ pacman -S --needed git bison ruby mingw-w64-i686-{toolchain,extra-cmake-modules,ruby,mruby,cegui,sfml,libxml++2.6,gperf}
+
+#### Optional dependencies ###
+
+The following packages are optional, you don't have to install these if you don't plan to generate installers or documentation:
+
+    $ pacman -S --needed mingw-w64-x86_64-{doxygen,graphviz,nsis,qt-installer-framework-git}
+
+For 32-bit:
+
+    $ pacman -S --needed mingw-w64-x86_64-{doxygen,graphviz,nsis,qt-installer-framework-git}
+
+Generating documentation also requires a few rubygems:
+
+    $ gem install adsf bundler coderay kramdown nanoc rdoc
+
+### CMake GUI Qt requirement workaround ###
+
+If you plan to use the CMake GUI, there's a few more steps required.
+The CMake GUI in msys2 is dynamically linked against Qt5 libraries. To use it, you need to have the Qt5 package installed:
+
+    $ pacman -S mingw-w64-x86_64-qt5
+
+But it's huge, it's going to take very long time to install and is really an overkill to install just to use CMake GUI. Instead, just use the command line CMake (as shown in the example below), or, if you still want to use the GUI, here's a little cheat:
+
+Check which version of CMake is currently installed (`pacman -Qs cmake`), go to [Cmake website][1] and download the same version for the appropriate architecture, and extract **ONLY** the executables (in the "bin" folder) to C:\msys64\mingw64\bin (or wherever you have installed msys2) and run the following command:
+
+    $ cp /mingw64/bin/mingw32-make.exe /mingw64/bin/make.exe
+
+
+### Building TSC ###
+
+From the same shell, that you installed packages for, run:
+
+    $ git clone git://github.com/Secretchronicles/TSC.git
+    $ cd TSC
+    $ git submodule init && git submodule update
```

Comment:
Sorry this is complete bikeshedding and useless, but: `git clone --recursive` does all of this, or even `git submodule update --init`.

Reply:
> Sorry this is complete bikeshedding and useless

I am grateful you are so friendly to me; these commands were most likely copied from our [current INSTALL.md](https://github.com/Secretchronicles/TSC/blob/devel/INSTALL.md#iv-installing-from-git) which was written by me.

Please, stay friendly, especially to newcomers. There is no need to react like this, and as this example shows, it might actually have side effect to react like that. You could long have corrected INSTALL.md yourself, by the way.

Reply:
I would definitely prefer to do those 3 lines with just 1. But, I'd still like the instructions to be consistent with the Linux ones. So, I'll leave for you to edit.